lib/ansible/modules/system/java_cert.py

```diff
@@ -92,6 +106,16 @@
   keystore_pass: changeit
   keystore_create: yes
   state: present
+
+# Import a pkcs12 keystore with a specified alias,
```

Comment:
All the examples seem wrong, as YAML only one example will be retained (because they all have the same key "java_cert" in the dictionary). Please modify all existing examples to a list:

```yaml
- name:  Import SSL certificate from google.com to a keystore, create it if it doesn't exist
  java_cert:
    cert_url: google.com
    keystore_path: /tmp/cacerts
    keystore_pass: changeit
    keystore_create: yes
    state: present
```